cf_xarray/accessor.py

```diff
@@ -584,6 +663,11 @@ def check_results(names, key):
         measures = []
         warnings.warn("Ignoring bad cell_measures attribute.", UserWarning)
 
+    try:
+        custom_criteria = OPTIONS["custom_criteria"][0]
+    except IndexError:
+        custom_criteria = OPTIONS["custom_criteria"]
```

Comment:
I don't see why we need `IndexError`. `OPTIONS['custom_criteria']` should always be a list.

We can use the `ChainMap` trick here too to go through all criteria.